debian/changelog

```diff
@@ -1,4 +1,4 @@
-sb-vision (0) unstable; urgency=low
+sb-vision (2018.2.0) unstable; urgency=low
```

Comment:
Are we going to do this to all the other packages too? I've no idea whether they depend on specific versions.

This **needs** testing before this merging.

Reply:
I've updated and tested this :+1:

Reply:
For our own future reference, could you detail somewhere what the testing procedure was?